internal/util/file.go

```diff
@@ -60,3 +60,18 @@ func GetExecPath(fileName string) (*string, error) {
 
 	return &path, nil
 }
+
+func ChownFile(file, owner string, permissions os.FileMode) error {
+	// Chown the file properly for the owner
+	uid, _ := GetUID(owner)
+	err := os.Chown(file, uid, -1)
+	if err != nil && os.Geteuid() == 0 {
```

Comment:
On Windows `os.Geteuid()` will always return `-1`, and this function will pass.  Is this a problem?

Reply:
> On Windows os.Geteuid() will always return -1, and this function will pass. Is this a problem?

_should_ not be as this is used mainly for setting cert file permissions which we do not do on windows